R/zchunk_LB123.LC_R_MgdPastFor_Yh_GLU.R

```diff
@@ -35,102 +35,314 @@ module_aglu_LB123.LC_R_MgdPastFor_Yh_GLU_DISABLED <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+    GCAM_commodity <- GCAM_region_ID <- value <- year <- Land_Type <- pasture_yield <-
+      frac <- total <- MgdPast <- MgdPast_adj <- veg_c <- VegVolume_m3m2 <- PopRatio <-
+      `mature age` <- GLU <- Yield_m3m2 <- Prod_bm3 <- MgdFor <- NULL   # silence package check notes
+
     # Load required inputs
-    L102.ag_Prod_Mt_R_C_GLU <- get_data(all_data, "L102.ag_Prod_Mt_R_C_GLU")
-    L102.ag_HA_bm2_R_C_GLU <- get_data(all_data, "L102.ag_HA_bm2_R_C_GLU")
     L108.ag_Feed_Mt_R_C_Y <- get_data(all_data, "L108.ag_Feed_Mt_R_C_Y")
-    L110.For_ALL_bm3_R_Y <- get_data(all_data, "L110.For_ALL_bm3_R_Y")
+    L110.For_ALL_bm3_R_Y <- get_data(all_data, "L110.For_ALL_bm3_R_Y") %>% unique
```

Comment:
Since you are working on improve the codes in that very first chunk, I have not touched anything there, but simply get the unique values here (see comments in #593).